examples/Rules-method-nextBest-NextBestProbMTDLTE.R

```diff
@@ -0,0 +1,41 @@
+# Example of usage for `NextBestProbMTDLTE` NextBest class.
+
+# Create the data.
+my_data <- Data(
+  x = c(0.1, 0.5, 1.5, 3, 6, 10, 10, 10),
+  y = c(0, 0, 0, 0, 0, 0, 1, 0),
+  ID = 1:8,
+  cohort = c(0, 1, 2, 3, 4, 5, 5, 5),
+  doseGrid = c(0.1, 0.5, 1.5, 3, 6, seq(from = 10, to = 80, by = 2))
+)
+
+# Initialize the CRM model used to model the data.
+my_model <- LogisticLogNormal(
+  mean = c(-0.85, 1),
+  cov = matrix(c(1, -0.5, -0.5, 1), nrow = 2),
+  ref_dose = 56
+)
+
+# Set-up some MCMC parameters and generate samples from the posterior.
+my_options <- McmcOptions(burnin = 100, step = 2, samples = 500)
+my_samples <- mcmc(my_data, my_model, my_options)
+
+# Define the rule for dose increments and calculate the maximum dose allowed.
+my_increments <- IncrementsRelative(
+  intervals = c(0, 20),
+  increments = c(1, 0.33)
+)
+next_max_dose <- maxDose(my_increments, data = my_data)
+
+# Define the rule which will be used to select the next best dose
+# based on the 'NextBestProbMTDLTE' class.
+prob_mtd_next_best <- NextBestProbMTDLTE(target = 0.33)
```

Comment:
```suggestion
nb_mtd_lte <- NextBestProbMTDLTE(target = 0.33)
```
just to keep the convention